src/dbnode/persist/fs/seek_manager.go

```diff
@@ -177,6 +199,7 @@ func NewSeekerManager(
 		bytesPool:                   bytesPool,
 		filePathPrefix:              opts.FilePathPrefix(),
 		opts:                        opts,
+		shardSet:                    sharding.NewEmptyShardSet(sharding.DefaultHashFn(1)),
```

Comment:
Do think we should set this to nil actually by default and then require it in the `Open` call. 

With other calls we should just make sure the `m.status != seekerManagerNotOpen` for any calls that might need to touch `m.shardSet`.